src/components/OpeningTimes.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+export default function OpeningTimes() {
+  const [openclose, setOpenclose] = useState("");
```

Comment:
Maybe you may not need to use state here, not sure which one is more efficient, using it or not :S I used plain js here: 

if (today >= 8 && today <= 17) {
    status = "open";
  } else {
    status = "closed";
  }